src/plugins/kbd.c

```diff
@@ -749,6 +749,19 @@ BDKBDZramStats* bd_kbd_zram_get_stats (const gchar *device, GError **error) {
 
 
 #ifdef WITH_BD_BCACHE
+
+gboolean wait_for_file (const char *filename) {
+    gint count = 500;
+    while (count > 0) {
+        g_usleep (100000); /* microseconds */
+        if (g_file_test (filename, G_FILE_TEST_EXISTS)) {
+            break;
+        }
+        --count;
+    }
+    return (count > 0) ? TRUE : FALSE;
```

Comment:
I think it would be nicer to do ``return TRUE`` instead of the ``break`` above and just ``return FALSE`` here.